apps/client/src/types/challenges.ts

```diff
@@ -7,6 +7,7 @@ export type ChallengeFormValues = {
 export type CardLayout = 'grid' | 'list';
 
 export interface Challenge {
+  id?: number;
```

Comment:
Why is `id` optional?

Reply:
Because you did not add it previously. So, I added it. Making it required

Reply:
Yes, it would be required.

```suggestion
  id: number;
```

Reply:
Done